samples/createJobFromPreset.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright 2020, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  projectId = process.env.GOOGLE_CLOUD_PROJECT,
```

Comment:
we can leave it just as projectId; we're trying to move away from using env variables in our samples. As long as you have a comment below, you should be fine!

Reply:
Refactored out.